NSS-Day-Cohort-69/tabloid-hamburgers
# Ft/post details

Added the ability to view a posts details

Added link to Post List page to navigate to the details of a post
Created postDetails component that renders required details (not styled af)


To Test:
log in and navigate to viewing all the posts
click on one of the Post Titles and see if you can see the details.
Try and view the post details for post 3 ...make sure nothing renders on the page. 


closes https://github.com/NSS-Day-Cohort-69/tabloid-hamburgers/issues/42
